tests/test_estimators.py

```diff
@@ -17,19 +19,26 @@
 from sknnr._base import IDNeighborsRegressor
 
 
-def get_kneighbor_estimator_instances() -> List[IDNeighborsRegressor]:
+def get_kneighbor_estimator_classes() -> List[Type[IDNeighborsRegressor]]:
```

Comment:
Good question! There were two reasons for this. 

1. When a parametrized test fails, pytest reports the parameters that were used. With instances, the names are pretty unhelpful, e.g. `estimator0`. With classes, it reports the class name instead, so it's easier to debug which test is failing.
2. `get_kneighbor_estimator_instances` apparently only gets called once by `pytest.mark.parametrize`, so each test gets passed the exact same instances. I don't think this was causing any issues here, but in `test_transformers` it was causing problems because one test would set the output, affecting the next test. By passing classes and instantiating in each test, we can make sure state doesn't persist between tests.

Reply:
Makes total sense on both counts.